### DISCUSSION TITLE
Support for select entity

### DISCUSSION THREAD

**Discussion by blaines** [zachowj/node-red-contrib-home-assistant-websocket#495] [Feature Request]
node-red-contrib-home-assistant-websocket currently does not appear to have support for the select entity.

https://www.home-assistant.io/integrations/select/

Are there any concerns with support for the select entity in Home Assistant?

**Comment by zachowj** [comment]
Support for creating them or acting on them?

**Reply by blaines** [reply]
I'm interested in both actually!
